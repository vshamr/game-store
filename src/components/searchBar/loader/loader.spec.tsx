import renderer from "react-test-renderer";
import Loader from "./index";

it("should render loader component", () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
