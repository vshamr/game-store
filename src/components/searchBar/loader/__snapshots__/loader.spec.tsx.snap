// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loader component 1`] = `
<div
  className="loader"
/>
`;
